tests/python-e2e/ops.py

```diff
@@ -25,6 +27,67 @@ def test_read_write_family(self):
             read_text = rw_file.readline()
             self.assertEqual(read_text, TEXT)
 
+    def test_fopen(self):
+        """
+        Opens a file in "/tmp/"
+        """
+        file_path, _ = self._create_new_tmp_file()
+        self.assertFalse(self._check_path_exists_on_host(file_path))
+        libc = CDLL('libc.so.6')
```

Comment:
would rather keep libc as a global value since its used in all functions

Reply:
I have no opposition!